src/server/editorServices.ts

```diff
@@ -708,7 +739,8 @@ function forEachAncestorProject<T>(
         if (
             !project.isInitialLoadPending() &&
             (
-                !project.getCompilerOptions().composite ||
+                (searchOnlyPotentialSolution && !project.getCompilerOptions().composite) ||
+                // TODO: Should this flag be shared between trying to load solution for find all references vs trying to find default project

```

Comment:
Is this still something you're looking to address?

Reply:
I really dont know if different option is needed or not. For now i will go with this as is and we can add option later if we see people asking for it?

Reply:
Will modify the comment to reflect that